courses/migrations/0021_backfill_paidcourserun_for_legacy_orders.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 3.2.15 on 2022-10-25 13:39
+
+from django.db import migrations
+
+
+def backfill_paidcourserun(apps, schema_editor):
+    """Backfill PaidCourseRun for mirgated/existing orders"""
+    PaidCourseRun = apps.get_model("courses", "PaidCourseRun")
+    CourseRun = apps.get_model("courses", "CourseRun")
+    Order = apps.get_model("ecommerce", "Order")
+    ContentType = apps.get_model("contenttypes", "ContentType")
+
+    for order in Order.objects.filter(state__in=["fulfilled", "review"]):
+        content_type = ContentType.objects.get_for_model(CourseRun)
+        for run_obj in order.lines.filter(purchased_content_type=content_type):
```

Comment:
This is not really a course-run object here in the loop. Wouldn't it be better to rename it as `order_line`?

Reply:
You are right. The variable name is a bit confusing, I think `purchased_line` or `order_line` works better than `run_obj`